test/native_objc_test/block_test.dart

```diff
@@ -21,6 +22,7 @@ void main() {
       final dylib = File('test/native_objc_test/block_test.dylib');
       verifySetupFile(dylib);
       lib = BlockTestObjCLibrary(DynamicLibrary.open(dylib.absolute.path));
+      generateBindingsForCoverage('block');
```

Comment:
Wait...so now you are generating the bindings but not checking that they are correct?